runtime/altair/Cargo.toml

```diff
@@ -146,312 +150,313 @@ substrate-wasm-builder = { workspace = true }
-  "cumulus-pallet-parachain-system/try-runtime",
-  "cumulus-pallet-xcm/try-runtime",
-  "cumulus-pallet-xcmp-queue/try-runtime",
-  "orml-asset-registry/try-runtime",
-  "orml-tokens/try-runtime",
-  "orml-xcm/try-runtime",
-  "orml-xtokens/try-runtime",
-  "pallet-anchors/try-runtime",
-  "pallet-aura/try-runtime",
-  "pallet-authorship/try-runtime",
-  "pallet-balances/try-runtime",
-  "pallet-base-fee/try-runtime",
-  "pallet-block-rewards/try-runtime",
-  "pallet-bridge/try-runtime",
-  "pallet-collator-allowlist/try-runtime",
-  "pallet-collator-selection/try-runtime",
-  "pallet-collective/try-runtime",
-  "pallet-crowdloan-claim/try-runtime",
-  "pallet-crowdloan-reward/try-runtime",
-  "pallet-democracy/try-runtime",
-  "pallet-elections-phragmen/try-runtime",
-  "pallet-ethereum/try-runtime",
-  "pallet-ethereum-transaction/try-runtime",
-  "pallet-evm/try-runtime",
-  "pallet-evm-chain-id/try-runtime",
-  "pallet-fees/try-runtime",
-  "pallet-foreign-investments/try-runtime",
-  "pallet-identity/try-runtime",
-  "pallet-interest-accrual/try-runtime",
-  "pallet-investments/try-runtime",
-  "pallet-keystore/try-runtime",
-  "pallet-liquidity-pools/try-runtime",
-  "pallet-liquidity-pools-gateway/try-runtime",
-  "pallet-liquidity-rewards/try-runtime",
-  "pallet-loans/try-runtime",
-  "pallet-membership/try-runtime",
-  "pallet-multisig/try-runtime",
-  "pallet-oracle-collection/try-runtime",
-  "pallet-oracle-feed/try-runtime",
-  "pallet-order-book/try-runtime",
-  "pallet-permissions/try-runtime",
-  "pallet-pool-fees/try-runtime",
-  "pallet-pool-registry/try-runtime",
-  "pallet-pool-system/try-runtime",
-  "pallet-preimage/try-runtime",
-  "pallet-proxy/try-runtime",
-  "pallet-restricted-tokens/try-runtime",
-  "pallet-restricted-xtokens/try-runtime",
-  "pallet-rewards/try-runtime",
-  "pallet-remarks/try-runtime",
-  "pallet-scheduler/try-runtime",
-  "pallet-session/try-runtime",
-  "pallet-sudo/try-runtime",
-  "pallet-swaps/try-runtime",
-  "pallet-timestamp/try-runtime",
-  "pallet-token-mux/try-runtime",
-  "pallet-transaction-payment/try-runtime",
-  "pallet-transfer-allowlist/try-runtime",
-  "pallet-treasury/try-runtime",
-  "pallet-uniques/try-runtime",
-  "pallet-utility/try-runtime",
-  "pallet-vesting/try-runtime",
-  "pallet-xcm/try-runtime",
-  "pallet-xcm-transactor/try-runtime",
-  "parachain-info/try-runtime",
+    # Enabling optional
+    "frame-try-runtime/try-runtime",
+    # Substrate related
+    "sp-runtime/try-runtime",
+    "frame-support/try-runtime",
+    "frame-system/try-runtime",
+    "frame-executive/try-runtime",
+    "fp-self-contained/try-runtime",
+    "polkadot-runtime-common/try-runtime",
+    # Locals
+    "cfg-primitives/try-runtime",
+    "cfg-traits/try-runtime",
+    "cfg-types/try-runtime",
+    "runtime-common/try-runtime",
+    "liquidity-pools-gateway-routers/try-runtime",
+    # Pallet list
+    "axelar-gateway-precompile/try-runtime",
+    "chainbridge/try-runtime",
```

Comment:
Curious about these changes. Are you using some code formatter for `.toml` files other than `taplo fmt`?

Reply:
I just used `taplo fmt` to fix the CI complaint. Also surprised it resulted in such a large diff.

Reply:
That weird. It also removes blank lines and increases the indentation. My `taplo` does not do that 😆. Which version are you using? Could you have some extra `.toml` plugin in your IDE?

Reply:
I don't have any extra toml plugin and I execute `taplo fmt` from an external terminal `iTerm` anyways. I updated to the newest version of taplo `v0.7.0` hoping to result in a smaller diff but that unfortunately that didn't happen.

Reply:
I'm using `0.9.0` and I think CI too

Reply:
For some reason it's fixed now 🤷